AuthenticationSimpleSample/AuthenticationSimpleSample/Auth/Constants.cs

```diff
@@ -10,7 +10,7 @@ public static class Constants
     {
         // All info can be found in your Azure AD B2C directory
         // AD
-        public static readonly string ClientId = "46ae553c-44c6-40bd-aded-daca74801f3e"; // from App Registrations 
+        public static readonly string ClientId = "ClientId"; // from App Registrations 
```

Comment:
I would suggest expanding on the comment here to more clearly instruct to the developer that they need to replace this string!